doc/content/examples/example9.md

```diff
@@ -0,0 +1,89 @@
+
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Response spectra of the individual soil response under the Ormsby Wavelet.
+
+### Soil Structure Interaction analysis
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. [fig:SSI_Amp] presents the amplification ratio as a function of the period. [fig:SSI_Amp_Valid] presents the amplification ratio obtained by [!citet](BolisettiThesis) using LS-Dyna. It is seen that the variation of the amplification ratios with period is similar between MASTODON and LS-Dyna.
+
+!listing examples/ex09/SSI_Response.i
+
+!media media/examples/ex09_SSI.png
+       style=width:80%;margin-right:0px;float:center;
+       id=fig:SSI_Amp
+       caption=Amplification ratio of the structural response spectra from the free-field soil in SSI analysis. This is computed using MASTODON.
+
+!media media/examples/ex09_SSI_Valid.png
+      style=width:80%;margin-right:0px;float:center;
+      id=fig:SSI_Amp_Valid
+      caption=Amplification ratio computed using LS-Dyna by [!citet](BolisettiThesis).
+
+[fig:SSI_FreeField] presents the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis. It is seen that these to spectra are very similar.
+
+!media media/examples/ex09_SSI_FreeField.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:SSI_FreeField
+       caption=Free-field response spectra from the SSI analysis and the individual soil analysis.
+
+## Note on the computational effort required
+
+When a $dt$ of 0.002$s$ is used, for a 3$s$ Ormsby Wavelet, this problem has 1500 time steps. It is not recommended to run this problem on your local machine and it is better to use a supercomputer. For details on how to use an INL supercomputer, please see [FAQ](help/faq.md). On FALCON machine with 1 full node, this problem takes about 3-3.5 hours to run.
```

Comment:
For this example, a INL supercomputer is used with dt of 0.002 sec for a total of 1500 time steps due to its expensive computational cost. For details on......